client/components/Balance/index.js

```diff
@@ -0,0 +1,33 @@
+import React, { PropTypes } from 'react';
+
+import formatAmount from 'utils/formatAmount';
+import './style.scss';
+
+const Balance = ({ title, symbol, amount, colorize }) => {
+  const formattedAmount = formatAmount(amount, false);
```

Comment:
I don't see any business logic here. It's just formatting but I agree that if we can save cycles on rendering then we should. I'll whip up a selector for this.